src/shared/mount-util.c

```diff
@@ -742,3 +745,265 @@ int mount_option_mangle(
 
         return 0;
 }
+
+int bind_mount_in_namespace(
+                pid_t target,
+                const char *propagate_path,
+                const char *incoming_path,
+                const char *src,
+                const char *dest,
+                int read_only,
+                int make_file_or_directory,
+                char **error_path) {
```

Comment:
Hm, why is this `error_path`? Maybe `error_message` would be clearer?

Reply:
Probably as I started from src/core/namespace.c and that's what the propagation variable is called. Changed.